ouroboros-consensus-cardano/app/snapshot-converter.hs

```diff
@@ -244,7 +246,7 @@ store config@Config{outpath = pathToDiskSnapshot -> Just (fs@(SomeHasFS hasFS),
       let h = V2.currentHandle lseq
       Monad.void $ V2.takeSnapshot ccfg nullTracer fs suffix writeChecksum h
     LMDB -> do
-      chlog <- newTVarIO (V1.empty state)
+      chlog <- newTVarIO (V1.empty (forgetLedgerTables state))
       lock <- V1.mkLedgerDBLock
       bs <- V1.newLMDBBackingStore nullTracer defaultLMDBLimits (V1.LiveLMDBFS tempFS) (V1.SnapshotsFS fs) (V1.InitFromValues (pointSlot $ getTip state) tbs)
```

Comment:
Here we are initializing an LMDB Backing store with a state that has tables. We should pass in the WriteHint so that we can `indexedPut` write the table.